cli/Makefile

```diff
@@ -26,6 +26,7 @@ release: gen-version
 	@cd build/src && zip -r ../${PROGNAME}.zip *
 	@echo '#!/usr/bin/env ${PYTHON}' | cat - build/${PROGNAME}.zip > build/${PROGNAME}
 	@chmod +x build/${PROGNAME}
+	@cd build && tar -czvf kubectl-kadalu.tar.gz kubectl-kadalu
```

Comment:
Small nit, I think it'll be good to use `${PROGNAME}` to be consistent?

Reply:
ack